docs/observabilityWalkthrough.md

```diff
@@ -0,0 +1,166 @@
+2. Search for the name of your storage account
+3. Click the CORS options on the menu on the left side:
+
+![cors menu option](./images/cors-menu.png)
+
+Add the following settings under **Table Service**:
+![cors settings](./images/cors-settings.png)
+
+**Note:** If you are running the service introspection spk dashboard in a port
+other than `4040`, add that entry in the settings instead.
+
+## Configure the Pipelines
+The Bedrock GitOps pipelines need to be configured to start sending data to
+`spk` service introspection. If you followed the guidelines from the (Prerequisites)[#prerequisites] each pipeline `yaml` will already have the script needed for introspection.
+
+To send data from Azure pipelines to the Azure Storage table created
+previously, a variable group needs to be configured in Azure DevOps (where the
+pipelines are).
+
+### Create a variable group
+Create the following `introspection-values.yaml` file with the variables:
+```
+name: "introspection-vg"
+description: "Service introspection values"
+type: "Vsts"
+variables:
+    INTROSPECTION_ACCOUNT_KEY:
+        value: "Set this to the access key for your storage account"
+        isSecret: true
+    INTROSPECTION_ACCOUNT_NAME:
+        value: "Set this to the name of your storage account"
+    INTROSPECTION_PARTITION_KEY:
+        value: "This field can be a distinguishing key that recognizea your source repository in the storage for eg. in this example, we're using the name of the source repository `hello-bedrock`"
+    INTROSPECTION_TABLE_NAME:
+        value: "Set this to the name of the table you created previously in [Create a table](#create-a-table)"
+```
+
+To configure the variable group run:
+
+```
+$ spk variable-group create --file introspection-values.yaml --org-name $ORG_NAME --devops-project $DEVOPS_PROJECT --personal-access-token $ACCESS_TOKEN
+```
+
+Where `ORG_NAME` is the name of the Azure Devops org, `DEVOPS_PROJECT` is the name of your Azure Devops project and `ACCESS_TOKEN` is the Personal access token associated with the Azure DevOps org. In [Setting up an HLD to Manifest pipeline](hldToManifestWalkthrough.md) we created a personal access token. 
+
+### Update the pipelines
+Next, we will update all the pipelines to include the variable group we created previously.
+
+```yaml
+variables:
+  - group: <your-variable-group-name>
+```
+
+First go to the pipelines on the Azure DevOps portal where you created your project:
+![Pipelines](images/pipelines.png)
+
+Next, click on the pipeline definition that you want to edit:
+![Edit pipeline](images/pipelines-edit.png)
+
+Add the name of the variable group you created in the file `introspection-values.yaml`. In this case, the name of the variable group we created was `introspection-vg`:
+![Edit pipeline](images/pipelines-edit-save.png)
+
+Repeat these steps for each pipeline definition.
+
+## Run the introspection tools
+Update the `spk-config.yaml` and populate the values for `introspection`.
```

Comment:
Do we give guidance anywhere that the user should retain this (can't remember if I have seen this or not)?

Reply:
The spk-config file is mentioned in the [hld to manifest walkthrough](https://github.com/microsoft/bedrock/blob/docs_spk/docs/hldToManifestWalkthrough.md)
I noticed that the [services](https://github.com/microsoft/bedrock/blob/docs_spk/docs/services.md) guideline only uses flags. Do we want to switch to flags across the guidelines for consistency? @timfpark let me know.